ChangeLog

```diff
@@ -1,3 +1,23 @@
+2020-04-26 Gregory John Casamento <greg.casamento@gmail.com>
+
+	* Headers/Additions/GNUstepGUI/GSTheme.h: Add declarations
+        for new drawing methods.
+	* Headers/AppKit/AppKit.h: Add new headers
+	* Headers/AppKit/NSAccessibilityProtocols.h
+	* Headers/AppKit/NSCell.h: Add NSControlStateValue enum
+	* Headers/AppKit/NSSwitch.h: Header for NSSwitch
+	* MISSING: Remove NSSwitch
+	* Source/GNUmakefile: Add .h and .m files
+	* Source/GSThemeDrawing.m: Add drawing code for knob and
+        switch basin.
+	* Source/GSXib5KeyedUnarchiver.m: Add decoding for
+        NSControlContents.  Make sure enabled is YES when not
+        present.
+	* Source/NSColor.m: Minor changes
+	* Source/NSMatrix.m: Remove uneeded code
```

Comment:
I think it is "unneeded" but you are the native speaker.
This occurs again two lines below.